client/source/class/cv/io/Client.js

```diff
@@ -491,7 +492,8 @@ qx.Class.define('cv.io.Client', {
           }
           if (options.listeners) {
             Object.getOwnPropertyNames(options.listeners).forEach(function(eventName) {
-              ajaxRequest.addListener(eventName, options.listeners[eventName], context);
+              var qxEventName = 'error' !== eventName ? eventName : 'statusError';
+              ajaxRequest.addListener(qxEventName, options.listeners[eventName], context);
```

Comment:
The 'statusError' listener is always added to ajaxRequest (see some lines below this one). What is the reason for this change? IMHO this change would result in two listeners to the 'statusError' event and no listener to the 'error' event.

Reply:
That's the other way round:
This line says: "please use `eventName` - except when it's `'error'`, then replace that with `'statusError'`"

There can be many listeners to `statusError`, one from the Client itself (line 507) and one per `options.listeners` at this line.

The reason for that change was that the listener for the `error` wasn't called, even when I made the server to give back a few 502 status codes